src/components/document-creation-modal.yml

```diff
@@ -0,0 +1,15 @@
+newDocument:
+  en: New document
+  de: Neues Dokument
+copyTitleSuffix:
+  en: (copy)
+  de: (Kopie)
+copySlugSuffix:
+  en: copy
+  de: kopie
+titleRequired:
+  en: Please specify a document title
+  de: Bitte geben Sie einen Dokument Titel an
```

Comment:
Dokumenttitel

Reply:
I tossed a coin between "Dokumenttitel", "Dokumentstitel" and "Dokument Titel" 🤦‍♀️ 
Thanks, corrected!